src/metabase/models/action.clj

```diff
@@ -396,3 +397,25 @@
 (defmethod serdes/storage-path "Action" [action _ctx]
   (let [{:keys [id label]} (-> action serdes/path last)]
     ["actions" (serdes/storage-leaf-file-name id label)]))
+
+;;;; ------------------------------------------------- Search ----------------------------------------------------------
+
+(search/define-spec "action"
+  {:model        :model/Action
+   :attrs        {:archived       true
+                  :collection-id  :model.collection_id
+                  :creator-id     true
+                  :database-id    :query_action.database_id
```

Comment:
@piranha Ngoc got a bit triggered by this kind of kebab / snake inconsistency.

Basically I'm using kebab everywhere in the schema, and snake in all the honey expressions. Then when data is selected from the db I leave it in snake too. This seems like the sweet spot between being idiomatic, not-too-magic, not-too-verbose, and not-too-much-data-transformation-overhead. 

wdyt?

Reply:
Just saw that, but I think it's fine - it's explicit enough not to be a problem, plus it will allow us to generate all the filters automatically with dashes (which is idiomatic for query strings).